.github/workflows/openmdao_test_workflow.yml

```diff
@@ -458,7 +461,7 @@ jobs:
           echo "============================================================="
           copy .coveragerc $HOME
           cd $HOME
-          testflo -n 2 openmdao --timeout=120 --show_skipped --coverage  --coverpkg openmdao --durations=20
+          testflo -n 1 openmdao --timeout=120 --show_skipped --coverage  --coverpkg openmdao --durations=20
```

Comment:
is there an issue with `-n 2` on Windows?  might want a comment here to document why...